lib/pbench/server/api/__init__.py

```diff
@@ -72,375 +61,23 @@ def create_app():
         sys.exit(1)
```

Comment:
If we are going to turn this into a utility function, we want to remove the calls to `sys.exit()`, and raise an exception that the outer layer can call `sys.exit()`.  See also #1955.

Reply:
yeah good point, will do